src/Elmish.WPF/ViewModel.fs

```diff
@@ -273,11 +274,15 @@ and [<AllowNullLiteral>] internal ViewModel<'model, 'msg>
       ( initialModel: 'model,
         dispatch: 'msg -> unit,
         bindings: Binding<'model, 'msg> list,
-        config: ElmConfig,
-        propNameChain: string)
+        bindingPerformanceLogThresholdMs: int,
+        propNameChain: string,
+        loggerFactory: ILoggerFactory)
       as this =
   inherit DynamicObject()
 
+  let log = loggerFactory.CreateLogger("Elmish.WPF.Bindings")
+  let logBindingPerformance = loggerFactory.CreateLogger("Elmish.WPF.BindingPerformance")
+
```

Comment:
I think the category names can be simplified to `Elmish.WPF` and `Elmish.WPF.Performance`.  The concept of "bindings" is implied since bindings for WPF are the value added by Elmish.WPF over Elmish....which is why there isn't a category called `Elmish.WPF.SomethingOtherThanBindings`.

Reply:
With my latest commit, there's `Elmish.WPF.Update` for `update` stuff: Errors, messages and updated models. Does that change your position on this?

In any case, I like `Elmish.WPF.Performance`, I'll change that.

Reply:
The events sent to the new category `Elmish.WPF.Update` are about an update by Elmish.  When I think about the concept of update for Elmish.WPF, I think of [`ViewModel<...>.UpdateModel(...)`](https://github.com/elmish/Elmish.WPF/blob/0ae868df7d590046899bd8b582bca174426481d2/src/Elmish.WPF/ViewModel.fs#L783).  I find the use of this category confusing.

Reply:
> When I think about the concept of update for Elmish.WPF, I think of [`ViewModel<...>.UpdateModel(...)`](https://github.com/elmish/Elmish.WPF/blob/0ae868df7d590046899bd8b582bca174426481d2/src/Elmish.WPF/ViewModel.fs#L783). I find the use of this category confusing.

`Elmish.WPF.Update` is a public concept. `UpdateModel` is an internal concept that users to not know about.

Still, I welcome other suggestions for category names.

Reply:
With my new perspective of Elmish as an implementation detail and our hiding of `Elmish.Program`, I like these category names.